lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["packages/*"],
-  "version": "1.61.0",
```

Comment:
do we need to update any contribution doc on how to version after this merge?

Reply:
same command is run